### DISCUSSION TITLE
Expanding on Gluten science

### DISCUSSION THREAD

**Discussion by fischerq** [hendricius/the-sourdough-framework#112] [Ideas]
@hendricius 
The section on gluten chemistry is quite short. 
E.g. [this article](https://www.seriouseats.com/what-is-gluten-free-bread-dough-pasta) has a few details that I found really interesting (on how glutenin and giandin work, interactions with salt). Also the reference to Harold McGee's [On Food and Cooking](https://www.amazon.com/dp/B000PAAH1W/) might be valuable.
I also think a table with numbers around protein content would be really useful, that helped me debug my flour (10% protein - low, 13% ~high, option of supplementing with pure gluten).

Would it be useful if I sent a pull request for some content around this?

**Comment by hendricius** [comment]
Sorry. I only saw this now. Definitely great idea.

I am still currently taking pictures for the inclusions chapter. Once this is ready I am happy to have a look and help on adding a PR. Will keep you posted. Thank you for everything so far!
